instance/tests/integration/integration_instance.py

```diff
@@ -107,6 +110,29 @@ def test_external_databases(self):
         self.assertTrue(instance.successfully_provisioned)
         self.assertFalse(instance.require_user_creation_success())
 
+    @shard(3)
```

Comment:
@bdero Currently, shard 0 finishes much faster than the other two shards (which run integration tests deplying an instance), so adding further tests to that shard (i.e. without any decorator) should be fine.  You can add one more container for free at https://circleci.com/gh/open-craft/opencraft/edit#parallel-builds if needed.